src/lib/modules/blockchain_connector/index.js

```diff
@@ -207,6 +209,25 @@ class BlockchainConnector {
     }
   }
 
+  addWeb3ToEmbarkJS() {
+    let code = '';
+    code += fs.readFileSync(utils.joinPath(__dirname, 'embarkjs.js')).toString();
+    code += "\nEmbarkJS.Blockchain.registerProvider('web3', __embarkWeb3);";
+
+    // TODO when we refactor code generator, refactor this to actually do something like connect
+    code += "\nEmbarkJS.Blockchain.setProvider('web3', {});";
```

Comment:
This looks a bit off. Seems like `setProvider()` options aren't optional: https://github.com/embark-framework/EmbarkJS/pull/31/files#r238181261.

I think we should adjust that.